model_compression_toolkit/core/common/hessian/trace_hessian_request.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List
+
+from enum import Enum
+
+from model_compression_toolkit.core.common import BaseNode
+
+
+class TraceHessianMode(Enum):
+    """
+    Enum representing the mode for trace Hessian's trace information computation.
+
+    This determines whether the Hessian's trace approximation is computed w.r.t weights or w.r.t activations.
+    Note: This is not the actual Hessian's trace but an approximation.
+    """
+    WEIGHTS = 0         # Hessian approximation based on weights
+    ACTIVATIONS = 1     # Hessian approximation based on activations
```

Comment:
Consider changing to "ACTIVATION" (no "S"), because this is like this is all our code (including enums).
For example: ActivationQauntizationMethod, all Activation quantizers, QuantizationTarget.Activation, etc.